### PR TITLE
VIRTS-776 Add obfuscator: base64 strip padding

### PR DESCRIPTION
**Pull request by rfrank27**

### CODE REVIEW

File: app/obfuscators/base64_no_padding.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            windows=['psh'],
+            darwin=['sh'],
+            linux=['sh']
+        )
+
+    def run(self, link, **kwargs):
+        link.command = link.command.replace('=', '')
+        return super().run(link)
+
+    @staticmethod
```
Comment (dhirst2880) [inline app/obfuscators/base64_no_padding.py]: I don't think we need static here, either that or the `self` argument should be removed. Same for the `psh` method below.

File: app/obfuscators/base64_no_padding.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    def sh(self, link, **kwargs):
+        return 'eval "$(echo %s=== | base64 --decode 2>/dev/null)"' % link.command
+
+    @staticmethod
+    def psh(self, link, **kwargs):
+        recoded = b64encode(link.command)
+        return '$string=%s;' +\
+            'while($string.Length %4 -ne 0) {$string="$string="};' +\
+            '$ExecutionContext.InvokeCommand.ExpandString(' +\
+            '[System.TextEncoding]::UTF8.GetString([convert]::FromBase64String($string)))' % recoded
```
Comment (ashinde7) [inline app/obfuscators/base64_no_padding.py]: I believe the Encoding class is under the System.TextEncoding namespace, so it is used like this : `System.Text.Encoding`
```suggestion
            '[System.Text.Encoding]::UTF8.GetString([convert]::FromBase64String($string)))' % recoded
```

File: app/obfuscators/base64_no_padding.py (hunk 1 above)
Comment (ashinde7) [reply app/obfuscators/base64_no_padding.py]: Done.

**Comment by dhirst2880** [comment]
Looks good to me. Onto @privateducky

**Pull request merged**